source/menus/important_flag_menu.cpp

```diff
@@ -36,7 +37,7 @@ void ImportantFlagMenu::draw() {
             break;
         case ENDLESS_NIGHT:
             setEventFlag(0x0A02);
-            break;
```

Comment:
Did you mean to remove this break statement?

Reply:
probably not